tools/pd-ut/testdata/group.1.etcdkey

```diff
@@ -0,0 +1,75 @@
+ get
```

Comment:
Are these files necessary? How will we maintain them?

Reply:
How about adding a script to generate it?

Reply:
It will be print after execute `./scripts/ci-subtask.sh {group_id}`. So we can see it in the github action 
<img width="1325" alt="image" src="https://github.com/user-attachments/assets/7023d0ea-490d-45b2-992a-993feb989f3a" />

BTW: the code is in https://github.com/okJiang/pd/blob/7025e1f6144f5f6e40546246e97b452e11d1329f/tools/pd-ut/ut.go#L419. If diff exists, it will also print https://github.com/okJiang/pd/blob/7025e1f6144f5f6e40546246e97b452e11d1329f/tools/pd-ut/ut.go#L421-L437


> How will we maintain them?

If the ci failed, we can copy the log to the xxxx.etcdkey